swarm.cabal

```diff
@@ -167,6 +167,7 @@ library
                       brick                         >= 1.5 && < 1.7,
                       bytestring                    >= 0.10 && < 0.12,
                       clock                         >= 0.8.2 && < 0.9,
+                      colour                        >= 2.3.6 && < 2.3.7,
```

Comment:
These version bounds seem overly restrictive.  At least we should put `< 2.4`, right?

Reply:
done